crates/service/src/database/mod.rs

```diff
@@ -2,7 +2,6 @@
 // SPDX-License-Identifier: Apache-2.0
 
 pub mod cost_model;
-pub mod dips;
```

Comment:
You removed dips from the modules but didn't remove `dips.rs` file. Is there any reason?

Reply:
I moved it to database.rs on the dips crate, so it shows in git as moved rather than removed

Reply:
You are right! Github tricked me